package.json

```
@@ -199,7 +199,8 @@
 		"typescript": "4.0.3",
 		"uuid": "7.0.2",
 		"wd": "1.12.1",
-		"webpack": "4.42.0",
+		"webpack": "5.1.3",
+		"webpack-cli": "4.1.0",
```

Flagging this one as safe to remove:

```suggestion
		"webpack-cli": "4.1.0",
```

Unless we will start using it explicitly in the project outside of `@wordpress/scripts`.

---

Good catch, I had to add it locally because I wanted to run webpack from command line 🙂

---

`webpack-cli` removed from `package.json`. Now it's only in `packages/scripts/package.json`.